data/random-teams.js

```diff
@@ -197,10 +197,10 @@ class RandomTeams {
 			if (species === 'Smeargle') {
 				pool = Object.keys(this.dex.data.Movedex).filter(moveid => !(this.dex.data.Movedex[moveid].isNonstandard || this.dex.data.Movedex[moveid].isZ || this.dex.data.Movedex[moveid].id === 'hiddenpower' && moveid !== 'hiddenpower'));
 			} else {
-				let lsetData = this.dex.getLearnsetData(template.speciesid);
+				let lsetData = this.dex.getLearnsetData(template.id);
 				// @ts-ignore
 				if (lsetData.exists) pool = Object.keys(lsetData.learnset).filter(moveid => lsetData.learnset[moveid].find(learned => learned.startsWith(this.gen.toString())));
-				if (template.species.substr(0, 9) === 'Necrozma-' || template.species.substr(0, 6) === 'Rotom-') {
+				if (template.name.substr(0, 9) === 'Necrozma-' || template.name.substr(0, 6) === 'Rotom-') {
```

Comment:
```suggestion
				if (template.name.startsWith('Necrozma-') || template.name.startsWith('Rotom-')) {
```

Wow, we still use `substr` in some places...